packages/utils/src/ts-eslint/SourceCode.ts

```diff
@@ -384,9 +384,7 @@ namespace SourceCode {
     visitorKeys: VisitorKeys | null;
   }
 
-  export interface VisitorKeys {
-    [nodeType: string]: string[];
-  }
```

Comment:
I do like the `Record` style more in general, but you can lose some nice implicit documentation sometimes (eg the variable name here).

I'm so torn about using a disable comment for this though because it's so minor.

Reply:
Yeah this is an interesting discussion. I'll undo the changes from `consistent-indexed-object-style` and file them as a todo to look at later, similar to `prefer-nullish-coalescing`.